regen-network/regen-web
# feat: edit profiles created with the Project Roles Form

## Description

Closes: regen-network/regen-registry#1727


---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

Create a new project or go to an existing project from https://deploy-preview-2100--regen-marketplace.netlify.app/profile/projects
Go to the Roles form and add a new profile for project developer or verifier.
You should then see the "edit profile" below the field, click and try to edit the profile.
Please be aware that "edit profile" won't get displayed for profiles that got added from the Roles form before this PR since we didn't assign a creator_id to those at the time. This has been added as part of this work to ensure that only the creator of a profile can edit it (until it's claimed by someone else who would log in using the profile wallet address if that was provided).

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
